src/etcetra/client.py

```diff
@@ -1033,6 +1034,135 @@ async def keys_range(
         )
         return [x.key.decode(encoding) for x in response.kvs]
 
+    async def election_campaign(
+        self,
+        name: str,
+        lease_id: int,
+        value: Optional[str] = None,
+        encoding: Optional[str] = None,
+    ) -> LeaderKey:
+        """
+        Campaign waits to acquire leadership in an election,
+        returning a LeaderKey representing the leadership if successful.
+        The LeaderKey can then be used to issue new values on the election,
+        transactionally guard API requests on leadership still being held,
+        and resign from the election.
+
+        Parameters
+        ---------
+        name
+            Name is the election’s identifier for the campaign.
+        lease_id
+            LeaseID is the ID of the lease attached to leadership of the election.
+            If the lease expires or is revoked before resigning leadership,
+            then the leadership is transferred to the next campaigner, if any.
+        value:
+            Value is the initial proclaimed value set when the campaigner wins the election.
+
+        Returns
+        -------
+        leader: etcetra.types.LeaderKey
+            Leader describes the resources used for holding leadereship of the election.
+        """
+        encoding = encoding or self.encoding
+        name_bytes = name.encode(encoding)
+        value_bytes: Optional[bytes] = value.encode(encoding) if value is not None \
+                                       else None
```

Comment:
Let's wrap it with braces to match our python coding style.

```suggestion
        value_bytes: Optional[bytes] = (
            value.encode(encoding) if value is not None
            else None
        )
```